tests/e2e/login/privacy-policy.spec.js

```diff
@@ -126,7 +127,13 @@ describe('Privacy policy', () => {
   });
 
   describe('for a french offline user', () => {
+    let passed = false;
     afterEach(async () => {
+      if (!passed) {
+        // I suspect this test is failing because of a conflict.
+        const userDoc = await utils.requestOnTestDb('/org.couchdb.user:offline?conflicts=true');
+        console.log(JSON.stringify(userDoc, null, 2));
```

Comment:
Wondering if there needs to be some clarification in the log incase someone looks. We have a decent amount of noise.

Reply:
Done.